packages/react-events/src/dom/Press.js

```diff
@@ -672,7 +672,10 @@ const pressResponderImpl = {
         if (previousPointerType !== pointerType) {
           return;
         }
-        if (type === 'pointermove' && activePointerId !== pointerId) {
+        if (
+          type === 'pointermove' &&
+          activePointerId !== nativeEvent.pointerId
```

Comment:
Why not extract pointerId above?

Reply:
The value is only used (and not `null`) within the the PointerEvent branches.